packages/nuxt/src/app/composables/state.ts

```diff
@@ -22,6 +22,18 @@ export function useState <T> (...args: any): Ref<T> {
   }
   const key = '$s' + _key
 
+  if (process.dev) {
+    const map = new Map<string, string>()
+    const defineInit = map.get(key)
```

Comment:
perf: you can likely avoid the multiple calls to `.toString()` below

```suggestion
    const defineInit = map.get(key)?.toString()
```